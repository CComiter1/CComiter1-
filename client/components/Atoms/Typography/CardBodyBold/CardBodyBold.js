import CardBody from '../CardBody/CardBody'

export default CardBody.extend`
  font-weight: bold;
`
