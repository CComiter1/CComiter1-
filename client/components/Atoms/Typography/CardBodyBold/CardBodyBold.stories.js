import React from 'react'
import { storiesOf } from '@storybook/react'

import CardBodyBold from './CardBodyBold'

storiesOf('Atoms/Typography', module)
  .add('CardBodyBold', () => <CardBodyBold>Ooowee</CardBodyBold>)
