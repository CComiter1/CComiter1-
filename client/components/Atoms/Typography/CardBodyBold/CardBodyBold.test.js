import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import CardBodyBold from './CardBodyBold'

describe('CardBodyBold', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <CardBodyBold>BOLD TEST</CardBodyBold>
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
