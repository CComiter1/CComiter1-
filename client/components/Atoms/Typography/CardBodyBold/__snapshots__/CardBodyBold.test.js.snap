// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBodyBold renders correctly 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
  font-weight: bold;
}

<p
  className="c0"
>
  BOLD TEST
</p>
`;
